charts/operator/Chart.yaml

```diff
@@ -5,10 +5,15 @@ icon: https://raw.githubusercontent.com/rancher/elemental/main/logo/icon-element
 version: 0.0.0
 appVersion: 0.0.0
 annotations:
+  catalog.cattle.io/auto-install: elemental-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Elemental
-  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.28.0-0'
```

Comment:
how/where did you define this range?

Reply:
This is a mandatory annotation for the Rancher Marketplace: since these annotations are for Rancher only, seems to me that makes sense to add all the required ones. 
Kubernetes 1.23 is the minimal version supported by Rancher 2.7.x and we require Rancher 2.7.x.
Kubernetes 1.28 is just an high Kube version: the value is arbitrary high and the common one used in the Rancher 2.7 marketplace.
I don't think this restriction will be put in place ever, in any case I wish that for Kube 1.28 users will install a newer chart version 😄